test/functional/test-error.js

```diff
@@ -464,7 +464,7 @@ describe('reportErrorToServer', () => {
     const e = user().createError('123');
     const data = getErrorReportData(undefined, undefined, undefined, undefined,
         e, true);
-    expect(data.exps).to.equal('test-exp=1,disabled-exp=0');
+    expect(data.exps).to.equal('inabox-rov=1,test-exp=1,disabled-exp=0');
```

Comment:
Looks like your Travis tests are failing, because 'inabox-rov=1' can appear sometimes, but not other times. So maybe:
```
if (data.exps.indexOf('inabox-rov=1') == -1 {
    expect(data.exps).to.equal('test-exp=1,disabled-exp=0');
} else {
    expect(data.exps).to.equal('inabox-rov=1,test-exp=1,disabled-exp=0');
}
I'll submit a PR against your branch w/ this

Reply:
When I tried to submit against your branch, it wanted to bring a lot of merge stuff in, too. To keep it simple, I just created a separate branch: https://github.com/ampproject/amphtml/pull/14359